kitsune/settings.py

```diff
@@ -407,7 +389,7 @@ def path(*parts):
 )
 
 # Make this unique, and don't share it with anybody.
-SECRET_KEY = '#%tc(zja8j01!r#h_y)=hy!^k)9az74k+-ib&ij&+**s3-e^_z'
+SECRET_KEY = config('SECRET_KEY')
```

Comment:
you should add some default value

Reply:
Nope. Not adding a default value makes it error on startup, which we want if no unique secret key is set. It's a security risk to check in a value that may accidentally be used in prod.